src/main.cpp

```diff
@@ -188,9 +189,24 @@ int main(int argc, char* argv[])
     spDebugConsole = nullptr;
     unsigned int startupAction = 0;
 
+#if defined(Q_OS_LINUX) && (QT_VERSION < QT_VERSION_CHECK(5, 12, 0))
```

Comment:
It is my belief that this will also be a factor on other non-MacOS and non-Windoze platforms - can you OR the `Q_OS_LINUX` with `Q_OS_FREEBSD`here and below please...! :pray:

Reply:
I have no idea how it is on FreeBSD, would rather not turn things on blindly without testing first! Let's get some feedback from FreeBSD users and then act on it.